content/docs/about/docs.md

```diff
@@ -38,6 +38,9 @@ GitHub.
 ## Contributing to the docs
 
 We welcome updates to the docs! Please help us make them better. Small fixes,
-typos, big fixes, plugging gaps&mdash;all are useful. For help with getting
-started, take a look at the 
-[README](https://github.com/kubeflow/website/blob/master/README.md).
+typos, big fixes, plugging gaps&mdash;all are useful. 
```

Comment:
Do you mean big or bug?

Reply:
Ha ha, I'm sure I meant "bug fixes" way back when. Good pickup! Fixed.